File: iniciante/tempodejogocomminutos1047.java
import java.io.IOException;
import java.util.*;

 

public class Main {
 
    public static void main(String[] args) throws IOException {
    
        Scanner scanner = new Scanner(System.in);

        int horaInicial = scanner.nextInt();
        int minutoInicial = scanner.nextInt();
        int horaFinal = scanner.nextInt();
        int minutoFinal = scanner.nextInt();

        System.out.println(horaCerta(horaInicial, minutoInicial, horaFinal, minutoFinal));
        
    }
    
    
       static String horaCerta(int a, int b, int c, int d) {

        int horas;
        int minutos;

        if (a == c && b == c && c == d) {
            horas = 24;
            minutos = 0;
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        }else if (a == c && b > d) {

            minutos = 60 - b + d;
            horas = 23;

            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        }else if (a == c && b < d) {
            horas = a - c;
            minutos = d - b;
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        } else if (a < c && b == d) {
            horas = c - a;
            minutos = 0;
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        } else if (a < c && b < d) {
            horas = c - a;
            minutos = d - b;
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        } else if (a > c && b < d) {
            horas = 24 - a + c;
            minutos = d - b;
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        } else if (a > c && b > d) {
            horas = 24 - a + c - 1;
            minutos = 60 - b + d;
           
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        } else if (a > c && b == d) {
            horas = 24 - a + c;
            minutos = b - d;
            if (minutos == 59){
                horas -= 1;
            }
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        }else if (a > c && b > d) {
            horas = 24 - a + c;
            minutos = 60 - b + d;
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        } else if (a < c && b > d) {
            horas = c - a - 1;
            minutos = 60 - b + d;
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";
        }else if (a == c && b == d) {
            horas = 24;
            minutos = 0;
            return "O JOGO DUROU " + horas + " HORA(S) E " + minutos + " MINUTO(S)";

        }
        return " ";
    }
 
}

File: iniciante/tempodejogocomminutos1047.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

function relogio(a, b, c, d){
    let horaInicio = a
    let minInicio = b
    let horaFinal = c
    let minFinal = d
    let countHora = 0
    let countMinuto = 0
    
    //Minutos
    while(true){
        minInicio++
        countMinuto++

        if(countMinuto == 60){
            countHora++
            countMinuto = 0
        }

        if(minInicio == minFinal){
            break
        }

        if(minInicio == 59){
            minInicio = -1
            horaInicio++
        }

    }
    
    //Horas
    while(true){
        if(horaInicio == horaFinal){
            break
        }

        horaInicio++
        countHora++
        
        if(horaInicio == 24){
            horaInicio = 0
        }

    }

    return `O JOGO DUROU ${countHora} HORA(S) E ${countMinuto} MINUTO(S)`
}

let entrada = lines.shift().split(' ')
let a = parseInt(entrada[0])
let b = parseInt(entrada[1])
let c = parseInt(entrada[2])
let d = parseInt(entrada[3])

console.log(relogio(a, b, c, d))
